src/main/java/jsr223/docker/compose/DockerComposeScriptEngineFactory.java

```diff
@@ -1,41 +1,42 @@
 package jsr223.docker.compose;
 
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import jsr223.docker.compose.utils.DockerComposeVersionGetter;
+import processbuilder.SingletonProcessBuilderFactory;
+import processbuilder.utils.ProcessBuilderUtilities;
 
 import javax.script.ScriptEngine;
 import javax.script.ScriptEngineFactory;
-
-import jsr223.docker.compose.utils.DockerComposeVersionGetter;
-import processbuilder.SingletonProcessBuilderFactory;
+import java.util.*;
 
 public class DockerComposeScriptEngineFactory implements ScriptEngineFactory {
 
-    private DockerComposeVersionGetter dockerComposeVersionGetter;
-
     // Script engine parameters
     private static final String NAME = "docker-compose";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "docker-compose" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1192)